spiceaidocs/docs/components/models/openai.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'OpenAI (or Compatible) Language Models'
+sidebar_label: 'OpenAI'
+sidebar_position: 4
+---
+
+To use a language model hosted on OpenAI (or compatible), specify the `openai` path in `from`. For a specific model, include it as the model id in `from` (see example below). Default `"gpt-3.5-turbo"`.
+Several params are specific to OpenAI models.
```

Comment:
```suggestion
These parameters are specific to OpenAI models:
```

**suggestion:** Re-words, because all of these parameters are OpenAI model specific?

If some aren't OpenAI specific, we should specify that in their description in lieu of `The OpenAI ...`.